src/vnsw/agent/vrouter/ksync/ksync_flow_memory.cc

```diff
@@ -11,9 +11,14 @@
 #include <fcntl.h>
 #include <sys/mman.h>
 #include <sys/types.h>
-//WINDOWS #include <sys/ipc.h>
-//WINDOWS #include <sys/shm.h>
-//WINDOWS #include <asm/types.h>
+#if !defined(_WIN32)
```

Comment:
I suggest you to use here ifndef like in the rest of the file

Reply:
Original file uses "#if defined" version (see lines: 6, 8).